vision/emergent_object/emergent_object.py

```diff
@@ -0,0 +1,61 @@
+import torch
+
+# You may have to install:
+#   pandas
+#   torchvision
+#   tqdm
+#   seaborn
```

Comment:
Dependencies should all be taken care of by #58 if you run in a poetry shell. These comments can be removed.